crates/orchestrator/src/network/parachain.rs

```diff
@@ -158,7 +158,12 @@ impl Parachain {
                     options.node_ws_url.as_str()
                 )
             })?;
-        let api = OnlineClient::<SubstrateConfig>::from_url(options.node_ws_url).await?;
+
+        let api = if subxt::utils::url_is_secure(&options.node_ws_url)? {
+            OnlineClient::<SubstrateConfig>::from_url(options.node_ws_url).await?
+        } else {
+            OnlineClient::<SubstrateConfig>::from_insecure_url(options.node_ws_url).await?
+        };
```

Comment:
I think there are two options here to make the code cleaner, 
1.  Receive a ref of the `node` as part of the options and call `api = node.wait_client()`
2. Create a helper that create a _client_ from the url, this could be in the helpers module.
Thx!

Reply:
Picked option 2 - using `get_client_from_url()` mentioned above.